akka-docs-dev/rst/java/stream-cookbook.rst

```diff
@@ -0,0 +1,14 @@
+.. _stream-cookbook-java:
```

Comment:
I was simply quieting down the sphinx warnings in order to improve the signal to noise ration.
Please fix as you seem fit.